doc/source/data/doc_code/quick_start.py

```diff
@@ -1,64 +1,119 @@
 # flake8: noqa
 # fmt: off
```

Comment:
Can we enforce flake8 here (especially helpful for line limits, to avoid horizontal wrap).